packages/sanity/src/structure/panes/document/DocumentEventsPane.tsx

```diff
@@ -24,7 +25,9 @@ export const DocumentEventsPane = (props: DocumentPaneProviderProps) => {
   const options = usePaneOptions(props.pane.options, params)
 
   const {selectedPerspectiveName} = usePerspective()
-  const {archivedReleases} = useReleases()
+  const {data} = useReleases()
+  const {archivedReleases} = useArchivedReleases(data)
```

Comment:
This doesn't seem correct given `useReleases` will not return the archived ones.
```suggestion
  const {data} = useAllReleases()
  const {archivedReleases} = useArchivedReleases(data)
```

Reply:
Good catch, this is definitely a mistake on my part (brought up [here](https://github.com/sanity-io/sanity/pull/8281#discussion_r1918817818))

Reply:
No longer needed since we [refactored it](https://github.com/sanity-io/sanity/pull/8281/commits/e06da423ba4b85a121741a6d06255383ba60b881) :)